packages/Alert/src/index.tsx

```diff
@@ -1,23 +1,30 @@
 import Button from "@blaze-react/button";
+import WithUtils from "@blaze-react/utils";
 import React, { Fragment, FunctionComponent, useState } from "react";
 interface IAlertProps {
   icon?: string;
   type?: string;
   close?: boolean;
-  children?: JSX.Element | string | {} | [];
+  utils: {
+    ternary: (condition: any, exprIfTrue: any, exprIfFalse?: any) => any;
+  };
+  children?: JSX.Element | string;
 }
 
 const Alert: FunctionComponent<IAlertProps> = ({
   children,
   close,
   icon,
   type,
+  utils: { ternary },
   ...attrs
 }): JSX.Element => {
-  const [offModal, setModalOff] = useState(false);
-  const assignType = type && `alert--${type}`;
-  const isDismissable = close && "alert--dismissable";
-  const withIcon = icon && "alert--icon";
+  const [offModal, setModalOff] = useState<boolean>(false);
+
+  const assignType = ternary(type, `alert--${type}`);
```

Comment:
use classnames package intead and if you want you can added it into utils

Reply:
done